src/app.js

```diff
@@ -0,0 +1,31 @@
+import express from "express";
+import * as bodyParser from "body-parser";
+import cors from "cors";
+import { decodeBase64 } from "bcryptjs";
+
+
+require('dotenv').config({ path: ".env"});
+const app = express();
+const APP_PORT = process.env.APP_PORT || 8080;
```

Comment:
You can still create a constant key in your .**env** for 8080;
Make sure you do not have plain values in your development.